Terratest/vpn/main.tf

```diff
@@ -0,0 +1,225 @@
+  delete_default_rules = true
+}
+
+resource "opentelekomcloud_networking_secgroup_rule_v2" "jumphost_secgroup_rule_eu_de_icmp" {
+  provider          = opentelekomcloud.de
+  direction         = "ingress"
+  ethertype         = "IPv4"
+  protocol          = "icmp"
+  remote_ip_prefix  = values(local.eu_nl_subnets)[0]
+  security_group_id = opentelekomcloud_networking_secgroup_v2.jumphost_secgroup_icmp_de.id
+
+  description = "Allow icmp traffic into the jumphost."
+}
+
+resource "opentelekomcloud_networking_secgroup_v2" "jumphost_secgroup_icmp_nl" {
+  provider             = opentelekomcloud.nl
+  name                 = "jumphost-eu-nl-icmp-sg"
+  description          = "Allow icmp traffic into the jumphost."
+  delete_default_rules = true
+}
+
+resource "opentelekomcloud_networking_secgroup_rule_v2" "jumphost_secgroup_rule_eu_nl_icmp" {
+  provider          = opentelekomcloud.nl
+  direction         = "ingress"
+  ethertype         = "IPv4"
+  protocol          = "icmp"
+  remote_ip_prefix  = values(local.eu_de_subnets)[0]
+  security_group_id = opentelekomcloud_networking_secgroup_v2.jumphost_secgroup_icmp_nl.id
+
+  description = "Allow icmp traffic into the jumphost."
+}
+
+module "jumphost_de" {
+  source = "../../modules/jumphost"
+  providers = {
+    opentelekomcloud = opentelekomcloud.de
+  }
+  node_storage_encryption_enabled = true
+  subnet_id                       = values(module.vpc_eu_de.subnets)[0].id
+  node_name                       = "jumphost-${var.context}-${var.stage}"
+  node_image_id                   = data.opentelekomcloud_images_image_v2.ubuntu_de.id
+  cloud_init                      = replace(join("\n", concat(["#cloud-config"], [for path in fileset("", "${path.root}/../../example_cloud_init/*.{yml,yaml}") : file(path)])), "{terraform_public_key}", tls_private_key.terraform_ssh_key.public_key_openssh)
+  additional_security_groups      = [opentelekomcloud_networking_secgroup_v2.jumphost_secgroup_icmp_de.name]
+}
+
+module "jumphost_nl" {
+  source = "../../modules/jumphost"
+  providers = {
+    opentelekomcloud = opentelekomcloud.nl
+  }
+  node_storage_encryption_enabled = true
+  subnet_id                       = values(module.vpc_eu_nl.subnets)[0].id
+  node_name                       = "jumphost-${var.context}-${var.stage}"
+  node_image_id                   = data.opentelekomcloud_images_image_v2.ubuntu_nl.id
+  cloud_init                      = replace(join("\n", concat(["#cloud-config"], [for path in fileset("", "${path.root}/../../example_cloud_init/*.{yml,yaml}") : file(path)])), "{terraform_public_key}", tls_private_key.terraform_ssh_key.public_key_openssh)
+  additional_security_groups      = [opentelekomcloud_networking_secgroup_v2.jumphost_secgroup_icmp_nl.name]
+}
+
+output "jh_de" {
+  value = module.jumphost_de.jumphost_address
+}
+
+output "jh_nl" {
+  value = module.jumphost_nl.jumphost_address
+}
+
+resource "time_sleep" "wait_180_seconds" {

```

Comment:
whenever you use the time-wait to delay execution, it is a good idea to add a comment why this is needed